app/Http/Controllers/TopicController.php

```diff
@@ -24,16 +25,30 @@ class TopicController extends ApiController
      */
     private function getMetaDataForModel(Topic $topic)
     {
+        if (!empty($topic->likes()->where('user_id', Auth::user()->id)->get()->first())) {
+            $isUser = true;
+            $likeId = $topic->likes()->where('user_id', Auth::user()->id)->get()->first()->id;
+            $countOfLikes=$topic->likes()->count();
```

Comment:
$countOfLikes=$topic->likes()->count();
please add spaces. Should be:
$countOfLikes = $topic->likes()->count();